packages/web/src/store/selectors/scanDisplayOptions/scanDisplayOptions.ts

```diff
@@ -46,14 +47,27 @@ export const makeSelectSortedAndFilteredScanPackages = () =>
   createSelector(
     [makeSelectScanPackagesByUrl(), makeSelectOptimizedScanDisplayOptions()],
     (packages = [], displayOptions) => {
-      const { packageFilters, packageSorters } = displayOptions;
+      const { searchText, packageFilters, packageSorters } = displayOptions;
 
-      const filteredPackages = packages.filter((pkg) =>
+      const searchValue = searchText.toLowerCase();
+
+      const matchedPackages = packages.filter(({ name, registryMetadata }) => {
+        return (
+          name.toLowerCase().includes(searchValue) ||
```

Comment:
This could probably be simplified as follows
```
[name, registryMetadata?.description?, ...].some(field => field?.toLowerCase().includes(searchValue))
```

Reply:
Just spreading, for example, `keywords` like this:
```js
[
  name,
  registryMetadata?.description,
  ...registryMetadata?.keywords
].some((field) => field?.toLowerCase().includes(searchValue));
```

Gives a error: `ESLint: Unsafe usage of optional chaining. If it short-circuits with 'undefined' the evaluation will throw TypeError`

So I came up with solution like this:
```js
const keywords = registryMetadata?.keywords?.length ? registryMetadata?.keywords : [];
const maintainers = registryMetadata?.maintainers?.length
  ? registryMetadata?.maintainers.map((it) => it.name)
  : [];

return [name, registryMetadata?.description, ...keywords, ...maintainers].some((field) =>
  field?.toLowerCase().includes(searchValue)
);
```

Is it better?

Reply:
Yep, this seems streamlined enough, LGTM